src/reducers/StakingReducer.ts

```diff
@@ -132,15 +125,6 @@ export const walletStakingStateReducer: Reducer<WalletStakingState, Action> = co
         return state
     }
   },
-  stakePlugins: (state = [], action: Action) => {
-    switch (action.type) {
-      case 'STAKING/UPDATE_PLUGINS':
-      case 'STAKING/SETUP':
```

Comment:
Ok, but delete the `stakePlugins` property from the `STAKING/SETUP` action payload.